library/src/androidTest/java/IntegrationUtils.kt

```diff
@@ -95,7 +95,12 @@ class IntegrationUtils {
             pa.commitActivationWithPassword(context, pin)
 
             // COMMIT ACTIVATION ON THE SERVER
-            makeCall<CommitObject>(body, "$cloudServerUrl/registration/commit")
+            val bodyCommit = """
+                {
+                  "externalUserId": "test"
+                }
+                """
```

Comment:
This keeps the whitespaces. Shouldn't there be something like  `.trimIndent()` at the end?

Reply:
Yop, added.